src/network/taraxa_capability.cpp

```diff
@@ -539,7 +547,9 @@ void TaraxaCapability::interpretCapabilityPacketImpl(NodeID const &_nodeID, unsi
       std::vector<Vote> next_votes;
       for (size_t i = 0; i < next_votes_count; i++) {
         Vote next_vote(_r[i].data().toBytes());
-        LOG(log_nf_next_votes_sync_) << "Received PBFT next vote " << next_vote.getHash();
+        auto next_vote_hash = next_vote.getHash();
```

Comment:
I'm curious why you broke this one line of code up into two?

Reply:
Don't need call next_vote.getHash() twice

Reply:
ok